internal/appsec/appsec.go

```diff
@@ -49,17 +75,21 @@ func Start(opts ...StartOption) {
 		opt(cfg)
 	}
 	appsec := newAppSec(cfg)
+
+	// Start the remote configuration client
+	log.Debug("appsec: starting the remote configuration client")
 	appsec.startRC()
 
-	// If the env var is not set ASM is disabled, but can be enabled through remote config
 	if !set {
-		log.Debug("appsec: %s is not set. AppSec won't start until activated through remote configuration", enabledEnvVar)
+		// AppSec is not enforced by the env var and can be enabled through remote config
+		log.Debug("appsec: %s is not set and won't start until activated through remote configuration", enabledEnvVar)
```

Comment:
```suggestion
		log.Debug("appsec: %s is not set. ASM won't start until activated through remote configuration", enabledEnvVar)
```
Otherwise it feels like we're saying the env var won't start. Feel free to replace `ASM` by anything else you deem more fitting

Reply:
I think appsec if better to use it the lib and the logs as this is our the prefix we use and ASM really is too ambiguous for devs.